radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/PeerManager.java

```diff
@@ -262,6 +263,10 @@ public void tryConnect(RadixNodeUri uri) {
       return PEER_BANNED.result();
     }
 
+    if (!this.config.peerWhitelist().contains(nodeId) && !this.config.peerWhitelist().isEmpty()) {
```

Comment:
I don't really like the empty allow list meaning everything is allowed -- or at least, it might make sense at configuration time, but I don't like this leaking out beyond the config class.

I quite like the idea of either separating it into two properties: `usePeerAllowList()` and `peerAllowList()` or just exposing `peerIsAllowed(nodeId)`

Reply:
I think I like the idea of a separate `usePeerAllowList` property more. I'll add it.